apps/explorer/lib/explorer/celo/util.ex

```diff
@@ -92,4 +92,24 @@ defmodule Explorer.Celo.Util do
   def epoch_by_block_number(bn) do
     div(bn, 17_280)
   end
+
+  def add_input_account_to_individual_rewards_and_calculate_sum(reward_lists_chunked_by_account, key) do
+    reward_lists_chunked_by_account
+    |> Enum.map(fn {:ok, rewards} -> rewards end)
```

Comment:
it's unusual to receive a list of `{:ok, rewards}` - this `:ok` should be tested and removed after whatever has created it has checked for it, otherwise the usefulness of checking return values is lost because you're not checking them when they've been returned